diffrax/_custom_types.py

```diff
@@ -1,9 +1,9 @@
 import typing
-from typing import Any, Optional, TYPE_CHECKING, Union
+from typing import Any, Literal, Optional, TYPE_CHECKING, TypeAlias, Union
```

Comment:
`TypeAlias` should be imported from `typing_extensions` for now, as we don't yet require Python 3.10.

Reply:
Aha, I see. Didn't know it was new in 3.10.